src/OpenLoco/src/Objects/VehicleObject.h

```diff
@@ -165,15 +165,15 @@ namespace OpenLoco
         uint8_t numAnimationFrames;      // 0x02
         uint8_t numCargoLoadFrames;      // 0x03
         uint8_t numCargoFrames;          // 0x04
-        uint8_t numRollFrames;           // 0x05
-        uint8_t halfLength;              // 0x06 the longest distance from pivot of body to either end of car component (not strictly body half length see crocidile train car)
+        uint8_t numTiltFrames;           // 0x05 currently the only valid values are 1 and 3
```

Comment:
i still feel roll is a better name than tilt. Tilt is train specific roll is the generic term.

Reply:
At present its only used for trains but I could see a future where we try get planes to roll.

Reply:
I think given the contentious naming of this variable we revert it for now